cloud/shared/bin/lib/destroy.py

```diff
@@ -7,28 +7,34 @@
 Destroy.py destroys the setup
 """
 
-###############################################################################
-# Load and Validate Inputs
-###############################################################################
-
-## Load the Config and Definitions
-config_loader = ConfigLoader()
-
-validation_errors = config_loader.load_config()
-if validation_errors:
-    new_line = '\n\t'
-    exit(
-        f"Found the following validation errors: {new_line}{f'{new_line}'.join(validation_errors)}"
-    )
-
-###############################################################################
-# Load Setup Class for the specific template directory
-###############################################################################
-
-template_dir = config_loader.get_template_dir()
-Destroy = load_destroy_class(template_dir)
-
-template_destroy = Destroy(config_loader)
-template_destroy.pre_terraform_destroy()
-terraform.perform_apply(config_loader, is_destroy=True)
-template_destroy.post_terraform_destroy()
+
+def main():
```

Comment:
Looks good modulo the comments from https://github.com/civiform/civiform/pull/3028#pullrequestreview-1060577560.